twisted/protocols/policies.py

```diff
@@ -651,7 +651,8 @@ def __init__(self, wrappedFactory, logfilePrefix, lengthLimit=None):
 
 
     def open(self, name):
-        return file(name, 'w')
+        import builtins
+        return builtins.open(name, 'w')
```

Comment:
So while I appreciate the idea, I don't actually think this does improve clarity. The fact is that `open` and `self.open` are evidently different and I don't believe there'd be any confusion caused by removing the import of builtins. However, late imports can often be a source of weird bugs, so I'd feel a lot better if we simply removed it.